tests/ctest_ut/ctestunittests.c

```diff
@@ -79,5 +79,14 @@ int main()
         }
     }
 
+    {
+        size_t temp_failed_tests = 0;
+        CTEST_RUN_TEST_SUITE(WindowsTypesTests, temp_failed_tests);
+        if (temp_failed_tests != 2) // 2 expected failing tests
+        {
+            failedTests++;
+        }
```

Comment:
isn't this missing the MSVC check?

Reply:
It is! Fixing it.

---
In reply to: [309833352](https://github.com/Azure/azure-ctest/pull/34#discussion_r309833352) [](ancestors = 309833352)